templates/base.html

```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
-<html lang="">
+<html id="htmltag" lang="{{ lang_code }}">
```

Comment:
Also if the other code is accepted change this:
```suggestion
<html lang="{{ lang_code }}">
```

Reply:
https://github.com/SpeedProg/eve-inc-waitlist/pull/12/commits/9f9443dea7c5098f7c8bea6aafe3037718fd7735 resolves this.